docs/content/NavList.mdx

```diff
@@ -0,0 +1,523 @@
+---
+title: NavList
+status: Draft
+description: Use nav list to render a vertical list of navigation links.
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+To render a horizontal list of navigation links, consider using [UnderlineNav](/UnderlineNav).
+
+## Examples
+
+### Simple
+
+```jsx
+<NavList>
+  <NavList.Item href="/" aria-current="page">
+    Dashboard
+  </NavList.Item>
+  <NavList.Item href="/pulls">Pull requests</NavList.Item>
+  <NavList.Item href="/issues">Issues</NavList.Item>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
+
+```html
+<nav>
+  <ul role="list">
```

Comment:
Do we need to render `role="list"` if we're using a `<ul>`?

Reply:
There's a bug in some browsers that removes the list semantics from `ul` if the element has `list-style: none` ([article](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)). `role="list"` restores the list semantics so it's properly announced by screenreaders. @alliethu or @ichelsea please correct me if I'm wrong 🙏

Reply:
Yes @colebemis! That's correct 🙂 . It's a failsafe for the browsers that remove it.